sbioapputils/app_runner/dev_utils.py

```diff
@@ -109,40 +113,40 @@ def validate_yaml_parameters(yaml_dict):
     
     if bad_formating:
         valid_check = False
-        print(f"These parameters are not formatted correctly in yaml: {bad_formating}. Please ensure no ':' values are included in keys and there is space between keys and values")
+        print(f"These parameters are not formatted correctly in yaml: {bad_formating}. Please ensure no ':' values are included in keys and there is space between keys and values. Also ensure that paths end with '/'")
         
     return valid_check
     
     
 def _define_files_from_yaml(yaml_dict):
     
     input_files = []
-    if yaml_dict['input_settings'].get('upload_options'):
-        for key in yaml_dict['input_settings']['upload_options'].keys():
-            if yaml_dict['input_settings']['upload_options'][key]['type'] == 'table':
+    if yaml_dict.get('input_settings'):
+        for key in yaml_dict['input_settings'].keys():
```

Comment:
same here with the keys. sorry I missed it in the previous review. all iterations where we use value should be done with `for key, value in dict.items()` and `value` should be used directly inside the loop